File: src/main/java/net/kigawa/kutil/unit/exception/NoFoundUnitException.java
package net.kigawa.kutil.unit.exception;

public class NoFoundUnitException extends RuntimeUnitException
{
    public NoFoundUnitException(String message, Throwable cause)
    {
        super(message, cause);
    }

    public NoFoundUnitException(String message)
    {
        super(message);
    }
}

File: src/main/java/net/kigawa/kutil/unit/exception/NoFoundUnitException.kt
package net.kigawa.kutil.unit.exception

@Suppress("unused")
class NoFoundUnitException: UnitException {
  constructor(unitClass: Class<*>, name: String?, message: String, cause: Throwable?):
          super(unitClass, name, message, cause)
  
  constructor(unitClass: Class<*>, name: String?, message: String): super(unitClass, name, message)
  constructor(unitClass: Class<*>, message: String, cause: Throwable?): super(unitClass, message, cause)
  constructor(unitClass: Class<*>, message: String): super(unitClass, message)
}